### PR TITLE
IA-4907: add POST endpoint for chartVersion

### PR DESCRIPTION
**Pull request by cpate4**

### CODE REVIEW

File: service/src/main/java/bio/terra/appmanager/controller/AdminController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  private final ChartService chartService;
+
+  public AdminController(ChartService chartService) {
+    this.chartService = chartService;
+  }
+
+  @Override
+  public ResponseEntity<Void> createChartVersions(List<ChartVersion> body) {
+
+    ;
```
Comment (jdcanas) [inline service/src/main/java/bio/terra/appmanager/controller/AdminController.java:21]: typo?

**Review by jdcanas — APPROVED** [review]
LGTM

File: service/src/main/resources/api/openapi.yml [hunk 2]
```diff
@@ -37,6 +37,29 @@ paths:
...
+      tags: [ admin ]
+      summary: Create new ChartVersion(s) to track
+      operationId: createChartVersions
+      security: [ ] # TODO: remove this to turn on "security"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ChartArray'
```
Comment (lucymcnatt) [inline service/src/main/resources/api/openapi.yml:52]: just curious about why this allows multiple chart versions, in my experience I thought the app developers only released one version at a time, so not sure when multiple is necessary?

**Review by lucymcnatt — COMMENTED** [review]
Generally LGTM, just had one question!

**Review by lucymcnatt — APPROVED** [review]

File: service/src/main/resources/api/openapi.yml (hunk 2 above)
Comment (cpate4) [reply service/src/main/resources/api/openapi.yml:52]: I was thinking of the use case where Sherlock (or other) is releasing multiple items at once. Happy to drop this back to being singles (it is an easy enough change either way). Happy to keep this simple at first and we can add on if need be. Thoughts?

Comment (lucymcnatt) [reply service/src/main/resources/api/openapi.yml:52]: ooh ok that makes sense, ie if we released new versions with the monolith or something, I'm down to keep it as is (and we can always iterate if we want to!)

Comment (cpate4) [reply service/src/main/resources/api/openapi.yml:52]: ... once the build is green I will squash-merge. thank you for talking it through with me - ☺️

**Pull request merged**